docker/production/.env.example

```
@@ -0,0 +1,68 @@
+
+# -----------------------------------------------------------------------------
+# Timezone (affects audit log timestamps and scheduled jobs)
+# -----------------------------------------------------------------------------
+TZ=America/Toronto
+
+# -----------------------------------------------------------------------------
+# Clinic identity (optional, shown in UI)
+# -----------------------------------------------------------------------------
+# Not currently substituted into carlos.properties by setup.sh, but reserved
+# for future use. Edit carlos.properties directly for UI customization.
+CLINIC_NAME=My Clinic
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)